src/Reports/SiteSummary.php

```diff
@@ -32,4 +32,16 @@ public function columns()
     {
         return singleton(Package::class)->summaryFields();
     }
+
+    public function getReportField()
```

Comment:
Please add a PHPDoc explaining what's happening here, even if it's just "Add a link out to the SilverStripe addons site" then use `{@inheritDoc}` to inherit the rest of the doc block